web/themes/custom/customtheme/components/01-atoms/buttons/back-to-top.js

```diff
@@ -0,0 +1,22 @@
+(function ($, Drupal, once) {
+  Drupal.behaviors.backToTop = {
+    attach: function (context, settings) {
+      var backButton = once('backToTopButton', '.font-container', context)[0];
+      $(backButton).hide();
+      $(window, context).on('scroll', function() {
+        var scrollPosition = $(window).scrollTop();
+
+        if (scrollPosition > 100) {
+          $(backButton).fadeIn();
+        } else {
+          $(backButton).fadeOut();
+        }
+      });
+
+      $(backButton).on('click', function (event) {
+        event.preventDefault();
+        $('html, body').animate({ scrollTop: 0 }, 1000);
+      });
+    },
+  };
+})(jQuery, Drupal, once);
```

Comment:
I would highly suggest to avoid using jQuery at all.
It doesn't do anything complicated here, so you are loading a full library on every page for the sake of a few functions.